pkg/minikube/sshutil/sshutil.go

```diff
@@ -61,6 +63,12 @@ func NewSSHClient(d drivers.Driver) (*ssh.Client, error) {
 	return client, nil
 }
 
+func TransferFile(f assets.CopyableFile, client *ssh.Client) error {
+	return Transfer(f, f.GetLength(),
+		f.GetTargetDir(), f.GetTargetName(),
+		f.GetPermissions(), client)
+}
+
```

Comment:
This is again updated from https://github.com/kubernetes/minikube/blob/2d2b484a22c389d257b6745675c29ddd264bb3d3/pkg/minikube/sshutil/sshutil.go#L95-L100